src/App.css

```diff
@@ -0,0 +1,60 @@
+@import url('https://fonts.googleapis.com/css2?family=Figtree:wght@300;400;500;600&family=Fondamento&display=swap');
```

Comment:
TODO: we should avoid using a CDN

From explorer https://github.com/search?q=repo%3Aoasisprotocol%2Fexplorer%20figtree&type=code :
```css
@import '../node_modules/@fontsource-variable/figtree/index.css';

font-family: "Figtree Variable"
```